packages/arb-token-bridge-ui/tests/e2e/specs/switchNetworks.cy.ts

```diff
@@ -109,26 +109,24 @@ describe('Switch Networks', () => {
 
     context('Test Networks list in Wrong Network UI', () => {
       it('should show wrong network UI', () => {
-        cy.changeMetamaskNetwork('kovan').then(() => {
+        cy.changeMetamaskNetwork('Sepolia test network').then(() => {
```

Comment:
The reason I changed this is because kovan is deprecated now and there's been issues trying to connect to it